massalabs/station
# 1286 remove acrylic dns from windows installer

## Delivery Quality Checklist

- [ ] **Breaking Changes in API:**
  Does this PR introduce breaking changes in the API?
    - If yes, have you considered making it backward compatible?
    - If backward compatibility is not considered, set the "breaking-change" label.

  - [ ] For bugfix PR, set the "bugfix" label
  - [ ] If this change should not appear in changelog, use "ignore-for-changelog" label

  Have you ensured that the version update by user is handled correctly?

- [ ] **PR Dependency:**
  Does this PR depend on another PR?
    - If yes, is it necessary for the dependency to be released prior to merging this one?

  - [ ] Are any necessary changes made to user-facing documentation?
  - [ ] Confirm that API documentation is updated with any relevant changes.
  - [ ] Check that README and other documentation files are accurate and current.

# QA Review
1. [Uninstall MS](https://docs.massa.net/docs/massaStation/uninstall#windows)
2. Download the installer from https://github.com/massalabs/station/actions/runs/7169232980?pr=1293
3. Install MS
4. Run MS, it should work fine (except WoC since `*.massa` is disable)
5. Open file explorer and copy paste the following path:
    ```
    %windir%\System32\drivers\etc\
    ```
6. Open the `hosts` file
7. You should see a line with `127.0.0.1 station.massa` 
8. You can close the file, but not the file explorer
9. Uninstall MS again
10. Reopen the `hosts` file
11. You should see no line mentionning `station.massa`
